src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -242,6 +242,10 @@ ov::Any py_object_to_any(const py::object& py_obj) {
         return py::cast<ov::frontend::type::Tensor>(py_obj);
     } else if (py::isinstance<ov::frontend::type::List>(py_obj)) {
         return py::cast<ov::frontend::type::List>(py_obj);
+    } else if (py::isinstance<ov::frontend::type::Str>(py_obj)) {
+        return py::cast<ov::frontend::type::Str>(py_obj);
+    } else if (py::isinstance<ov::frontend::type::PyNone>(py_obj)) {
+        return py::cast<ov::frontend::type::PyNone>(py_obj);
```

Comment:
So we continue affecting the common part, but the changes are required for (at this moment) for PT FE only. Any way to decompose functionality of this function and distribute it to more specialized parts like pt fe py bindings?

Reply:
Not sure how to do it. Those types should be shared between frontends though, so I don't think that this is huge issue